test/kitchen/tasks/run-test-kitchen.sh

```diff
@@ -83,15 +83,36 @@ if [ -z ${SERVER_PASSWORD+x} ]; then
   export SERVER_PASSWORD=$(< /dev/urandom tr -dc A-Za-z0-9 | head -c32)
 fi
 
-cp kitchen-azure.yml kitchen.yml
+if [[ $# == 0 ]]; then
+  echo "Missing run suite argument. Exiting."
+  exit 1
+fi
+
+cp kitchen-azure-common.yml kitchen.yml
 
 ## check to see if we want the windows-installer tester instead
-if [[ $#  != 0 && $1 == "windows-install-test" ]]; then
-  cp kitchen-azure-winstall.yml kitchen.yml
+if [[ $1 == "windows-install-test" ]]; then
+  cat kitchen-azure-winstall.yml >> kitchen.yml
```

Comment:
I guess there is no problem repeating `suites:` in the YML?

Reply:
There can be only one `suites:` param in the yaml, since only one of the files will get appended: either `kitchen-azure-winstall.yml`, `kitchen-azure-chef-test.yml`, `kitchen-azure-step-by-step-test.yml`, `kitchen-azure-install-script-test.yml`, `kitchen-azure-upgrade5-test.yml`, or `kitchen-azure-upgrade6-test.yml`, and the base file (`kitchen-azure-common.yml`) doesn't contain any `suites:` param.